zenodo/modules/records/serializers/schemas/datacite.py

```diff
@@ -422,6 +422,8 @@ class DataCiteSchemaV4(DataCiteSchema):
 
     contributors = fields.Method('get_contributors')
 
+    descriptions = fields.Method('get_descriptions')
```

Comment:
Since `descriptions` is already defined in the `DataCiteSchema` class, this is not necessary